### PR TITLE
chore: ei-2921 - updating dependencies for vuln fixes

### PR DESCRIPTION
**Pull request by conbon**
Depends on [grpc-client-rs](https://github.com/Iotic-Labs/iotics-grpc-client-rs/pull/43)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -25,8 +24,10 @@ tokio = { version = "1.21", features = ["macros", "rt-multi-thread", "time"] }
 
 # use this if you want to be able to change both repos in the same time
 # iotics-grpc-client = { path = "../iotics-grpc-client-rs" }
-iotics-grpc-client = "4.0"
+# iotics-grpc-client = "4.0"
+iotics-grpc-client = { git = "https://github.com/Iotic-Labs/iotics-grpc-client-rs.git", branch="ei-2921" }
```
Comment (conbon) [inline Cargo.toml]: update to release version of grpc-client

File: Cargo.toml [hunk 2]
```diff
@@ -25,8 +24,10 @@ tokio = { version = "1.21", features = ["macros", "rt-multi-thread", "time"] }
...
 
 # use this if you want to be able to change both repos in the same time
 # iotics-identity = { path = "../iotics-identity-go/ffi/rust" }
 iotics-identity = { git = "https://github.com/Iotic-Labs/iotics-identity-go.git" }
+dotenvy = "0.15.7"
```
Comment (vtermanis) [inline Cargo.toml]: Should this be above, replacing the `dotenv` line?

Comment (conbon) [reply Cargo.toml]: I used cargo remove / add and left it where it was added. I can move it up if you want though

Comment (vtermanis) [reply Cargo.toml]: .. just thinking that it fits more with the layout before (3rd-party first) - but this is minor - up to you

**Review by vtermanis — APPROVED** [review]

**Pull request merged**